### DISCUSSION TITLE
Duda, Help meeee ...!!

### DISCUSSION THREAD

**Discussion by kmn04MX** [Launch-X-Latam/MisionFrontEnd#592] [Team Posts]
Holaaa, espero alguien pueda ayudarme a contestar.
Alguien sabe hasta cuando tenemos tiempo para poder ver el material de innovación de backend?
Les agradecería si alguien pudiera ayudarme. :(

**Comment by Drako9159** [comment]
Estuve en la edición pasada que terminó como en Julio y todavía puedo acceder al material, supongo que a esta le queda bastante tiempo-

**Comment by VazRos03** [comment]
Estoy igual, apenas acabo de terminar el material de react :c y me preocupa que cierren el material y aún no acabe de verlo

**Comment by kmn04MX** [comment]
Voy atrasado y será imposible para lo del hackeaton, es mas, ni sé cuando es para empezar, lo importante es el conocimiento y es lo que me interesa. Muchas gracias por contestarme.

**Comment by kmn04MX** [comment]
> Estoy igual, apenas acabo de terminar el material de react :c y me preocupa que cierren el material y aún no acabe de verlo

Aún hay tiempo por lo que dice @Drako9159
